DOCS-856: Update line follower code (#1636)
viamrobotics/docs: docs/tutorials/services/webcam-line-follower-robot.md

   Then click **Create Service**.

   In your vision service’s panel, select the color your vision service will be detecting, as well as a hue tolerance and a segment size (in pixels).

>    Use a color picker like [colorpicker.me](https://colorpicker.me/) to approximate the color of your line and get the corresponding hex or rgb value to put in your config.

Author: ```suggestion
   Use a color picker like [colorpicker.me](https://colorpicker.me/) to approximate the color of your line and get the corresponding rgb value.
```

>    Use a color picker like [colorpicker.me](https://colorpicker.me/) to approximate the color of your line and get the corresponding rgb value.
>    We used `rgb(25,255,217)` to match the color of our green electrical tape, and specified a segment size of 100 pixels with a tolerance of 0.06, but you can tweak these later to fine tune your line follower.

Reviewer: ```suggestion
   We used `#19ffd9` to match the color of our green electrical tape, and specified a segment size of 100 pixels with a tolerance of 0.06, but you can tweak these later to fine tune your line follower.
```
I think since both the builder color picker, and the raw JSON, both use hex, it should say this. And I don't think we need to mention the external color picker anymore since there's the builtin one.

>    Use a color picker like [colorpicker.me](https://colorpicker.me/) to approximate the color of your line and get the corresponding rgb value.
>    We used `rgb(25,255,217)` to match the color of our green electrical tape, and specified a segment size of 100 pixels with a tolerance of 0.06, but you can tweak these later to fine tune your line follower.

Author: ```suggestion
   Use a color picker like [colorpicker.me](https://colorpicker.me/) to approximate the color of your line and get the corresponding rgb or hex value.
   We used `rgb(25,255,217)` or `#19FFD9` to match the color of our green electrical tape, and specified a segment size of 100 pixels with a tolerance of 0.06, but you can tweak these later to fine tune your line follower.
```

>    We used `#19FFD9` (or `rgb(25,255,217)`) to match the color of our green electrical tape, and specified a segment size of 100 pixels with a tolerance of 0.06, but you can tweak these later to fine tune your line follower.

Author: ```suggestion
   We used `rgb(25,255,217)` to match the color of our green electrical tape, and specified a segment size of 100 pixels with a tolerance of 0.06, but you can tweak these later to fine tune your line follower.
```

   In your vision service’s panel, select the color your vision service will be detecting, as well as a hue tolerance and a segment size (in pixels).

>    Use a color picker like [colorpicker.me](https://colorpicker.me/) to approximate the color of your line and get the corresponding hex code to put in your config.

Author: Updated

    print("connected")
    camera = Camera.from_robot(robot, "my_camera")
    base = Base.from_robot(robot, "scuttlebase")

    # Put your detector name in place of "green_detector"
    green_detector = VisionClient.from_robot(robot, "green_detector")

>     detections = await green_detector.get_detections_from_camera(camera)
>     print(detections)

Author: ```suggestion
```
